jetty-core/jetty-http3/jetty-http3-qpack/src/main/java/org/eclipse/jetty/http3/qpack/internal/table/DynamicTable.java

```diff
@@ -164,9 +156,24 @@ public Entry get(HttpField field)
         return _fieldMap.get(field);
     }
 
+    public Entry getRelative(int index)
+    {
+        return getRelative(index, getInsertCount());
+    }
+
+    public Entry getRelative(int index, int base)
+    {
+        return getAbsolute(base - 1 - index);
+    }
+
+    public Entry getPostBase(int index, int base)
```

Comment:
Can you switch the parameters order to be `base, index`? Feels more natural.

Reply:
done